### PR TITLE
Journey retreive data from cancer api

### LINKED ISSUE #171: Change persona images to background-image(should take up half of the screen)

### PR DESCRIPTION
**Pull request by fdenoodt**
Closes #171

### CODE REVIEW

File: frontend/src/assets/css/journey/comparison.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.compContentContent {
+  margin-top: 7em;
+  text-align: right;
+  /* max-width: 20em; */
+  /* padding-right: 2em; */
+}
+
+.compContentContent h1 {
+  /* padding-right:  */
```
Comment (archived-m) [inline frontend/src/assets/css/journey/comparison.css:19]: What's with this double naming?
`compContentContent`
`compVisVis`

Comment (fdenoodt) [reply frontend/src/assets/css/journey/comparison.css:19]: div in div

Comment (archived-m) [reply frontend/src/assets/css/journey/comparison.css:19]: It's still bad :')

File: frontend/src/assets/css/main.css [hunk 2]
```diff
@@ -2,20 +2,28 @@
...
+} */
+
 body {
     margin: 0;
     padding: 0;
     font-family: 'Montserrat', sans-serif;
 }
 
 h1, h2, h3, h4, h5, h6 {
+    font-size: 25pt;
```
Comment (archived-m) [inline frontend/src/assets/css/main.css]: Please don't set title font-sizes globally. Different titles need different sizes

**Review by archived-m — CHANGES_REQUESTED** [review]
When a user skips the questions at the start of the journey, the calculation for everything becomes either `NaN` or `loading`.

As discussed - if a question is skipped, generate a random persona so people can go through the journey with actual data

**Review by archived-m — COMMENTED** [review]
General note: make commit messages descriptive. Make sure they describe the change you just made.

File: frontend/src/assets/css/journey/comparison.css (hunk 1 above)
Comment (fdenoodt) [reply frontend/src/assets/css/journey/comparison.css:19]: i'll change soon.

File: frontend/src/assets/css/main.css (hunk 2 above)
Comment (fdenoodt) [reply frontend/src/assets/css/main.css]: something left from Q, i don't think it actually did something.
Will delete

**Review by archived-m — APPROVED** [review]

**Pull request merged**